clients/drcachesim/tests/invariant_checker_test.cpp

```diff
@@ -52,19 +52,26 @@ class checker_no_abort_t : public invariant_checker_t {
     {
     }
     std::vector<std::string> errors;
+    std::vector<memref_tid_t> error_tids;
+    std::vector<uint64_t> error_refs;
```

Comment:
Maybe add a comment to say that these refs should be interpreted as ordinals of the per-thread memrefs where some error was found.

Reply:
Done.